pkg/operation/common/types.go

```diff
@@ -534,6 +540,10 @@ const (
 	PrometheusTLS = "prometheus-tls"
 	// KibanaTLS is the name of the secret resource which holds the TLS certificate for Kibana.
 	KibanaTLS = "kibana-tls"
+	// EtcdServerTLS is the name of the secret resource which holds TLS server certificate of Etcd
+	EtcdServerTLS = "etcd-server-cert"
+	// EtcdClientTLS is the name of the secret resource which holds TLS client certificate of Etcd
+	EtcdClientTLS = "etcd-client-tls"
```

Comment:
The server tls secret is called `etcd-server-cert`. Why not calling this `etcd-client-cert`?

Reply:
The client cert `etcd-client-tls` is already created and is not updated. We are only changing the `etcd-server-tls` secret. `etcd-server-tls` is deleted and `etcd-server-cert` is created in its place.